tests/meta/upload/scout/conftest.py

```diff
@@ -598,6 +644,19 @@ def rnafusion_analysis_obj(analysis_obj: Analysis) -> Analysis:
     return analysis_obj
 
 
+@pytest.fixture
+def tomte_analysis_obj(analysis_obj: Analysis) -> Analysis:
+    """Return a tomte analysis object."""
```

Comment:
```suggestion
    """Return a Tomte analysis object."""
```

Reply:
It also sets values making into a Tomte analysis (not simply fetching). Would be nice to reflect in the doc str.